docs/introduction.md

```diff
@@ -1,39 +1,52 @@
 ---
 title: Introduction
-sidebar_position: 1
+description: Welcome to the Secure Customer developer documentation for Beyond Identity. Here you'll find guides for building strong authentication across your applications using public-private key pairs (what we call universal passkeys). 
+keywords:
+  - secure customer 
+  - developer documentation
+  - how it works
+last_update:
+  date: 01/12/2023
+  author: Patricia McPhee
 ---
 
-Welcome to the developer documentation for Beyond Identity. This is where you can find guides for building authentication across your applications using universal passkeys. 
+Welcome to the Secure Customer developer documentation for Beyond Identity. Here you'll find guides for building strong authentication across your applications using *universal passkeys* (public-private key pairs). 
+
 
 If you run into any issues or have feedback for us along the way, you can let us know by **[joining our Slack community](https://join.slack.com/t/byndid/shared_invite/zt-1anns8n83-NQX4JvW7coi9dksADxgeBQ)**.
 
-## Just looking to get going? 
+## Are you looking to get going quickly?
+
+Follow the [Getting Started guide](./getting-started), where you'll create and configure your Beyond Identity account and connect it to a sample application. This application demonstrates common workflows for authentication. After working with the sample application, you can ...
+
+
+## What is Beyond Identity?
 
-Follow our [Getting Started guide](/docs/v1/getting-started). 
 
-## What is Beyond Identity
+Beyond Identity is a new kind of authenticator that advances your security posture. Instead of passwords, users download the Beyond Identity Authenticator to set up their secure, device-bound credentials for zero-friction [passwordless authentication](https://www.beyondidentity.com/resources/passwordless-authentication).
```

Comment:
Hi! I realize this is just a draft but I was reading the PR out of curiosity! We do have an authenticator that users download but this is more for Secure Workforce. We also offer SDKs that embedded the authenticator into applications so the user doesn't need to physically download an authenticator in that case, which is Secure Customer. And I think we're going with the term "passkey" instead of "credential" going forward because we have been FIDO certified.

Reply:
You are correct that this is a draft and I am reworking a lot of the content for better user flow as per Nelson and Colton's request.  This is a ROUGH DRAFT.   I will close this PR.